src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -568,8 +568,8 @@ public void testDestroyCheckerWithWrongCacheFileNameLength() throws Exception {
         }
         catch (IllegalStateException ex) {
             assertWithMessage("Cause of exception differs from IOException")
-                    .that(ex.getCause() instanceof IOException)
-                    .isTrue();
+                    .that(ex.getCause())
```

Comment:
This also can be
```java
.that(ex)
.hasClauseThat()
  .isInstanceOf(IOException.class);
```

Reply:
this is one of case where new syntax does not make it better
it become just different and more verbose with no value.